### PR TITLE
fix: cfd-566 fix netex output

### PR DESCRIPTION
**Pull request by JwanKhalaf**
# Description

-   When the first stop in the first fare stage matches the last stop in the last fare stage, the said stop is removed.

# Testing instructions

-   Test with a route that matches above description

# Type of change

-   [ ] feat - A new feature
-   [x] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

### CODE REVIEW

File: repos/fdbt-site/src/interfaces/index.ts [hunk 1]
```diff
@@ -583,21 +583,13 @@ export interface JourneyPattern {
...
     };
+
     endPoint: {
         Id: string;
         Display: string;
     };
-    stopList: string[];
-}
 
-export type StopPoint = {
```
Comment (RobCatton) [inline repos/fdbt-site/src/interfaces/index.ts:593]: Were these unused?

File: repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts [hunk 2]
```diff
@@ -29,8 +29,41 @@ export const getUniquePriceGroups = (fareZones: FareZone[]): string[] => [
 
 export const getIdName = (name: string): string => name.replace(/(\s)+/g, '_');
 
-export const getPointToPointScheduledStopPointsList = (fareZones: FareZone[]): ScheduledStopPoints[] =>
-    getStops(fareZones).map(stop => ({
+export const existsTwiceInArray = (array: string[], element: string): boolean => {
```
Comment (RobCatton) [inline repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts:32]: Should we get rid of this as the 'hasDuplicates' function is the one we use?

File: repos/fdbt-site/src/interfaces/index.ts (hunk 1 above)
Comment (JwanKhalaf) [reply repos/fdbt-site/src/interfaces/index.ts:593]: This particular one was, there was the exact same thing defined elsewhere.

File: repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts (hunk 2 above)
Comment (JwanKhalaf) [reply repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts:32]: Good catch, I forgot to remove that.

**Review by RobCatton — APPROVED** [review]

**Pull request merged**